fwprovider/nodes/apt/repo_test.go

```diff
@@ -396,62 +308,42 @@ func TestAccResourceStandardRepoValidInput(t *testing.T) {
 			Steps: []resource.TestStep{
 				// Test the "Create" and "Read" implementations.
 				{
-					Config: fmt.Sprintf(
-						`
-					resource %q %q {
-						%s = %q # handle
-						%s = %q # node
-					}
-					`,
-						strings.Split(testAccResourceStandardRepoSelector, ".")[0],
-						strings.Split(testAccResourceStandardRepoSelector, ".")[1],
-						apt.SchemaAttrNameStandardHandle, testAccResourceStandardRepoHandle,
-						apt.SchemaAttrNameNode, te.NodeName,
-					),
+					// 	PUT /api2/json/nodes/{node}/apt/repositories with handle = "no-subscription" will create a new
+					// entry in /etc/apt/sources.list on each call :/
```

Comment:
Oh, this is …interesting. I guess this is the reason why I had flaky test runs while implementing the resource.